api.planx.uk/src/app/index.ts

```diff
@@ -0,0 +1,13 @@
+// The middleware of an Express app is excecuted in sequence.
+// This means that the order of `app.use(...)` statements is significant.
+//
+// This file exposes the express app after middlewar has been applied in a meaningful order:
+//
+//   1. init       - this sets up app configuration for all routes
+//   2. cookieless - this adds routes that are available without cookies
+//   3. secure     - this adds authenticated routes
+//   4. error      - this adds catchall middleware for error handling
```

Comment:
Appreciate the documentation here 👍 

I'm not _exactly_ sure of the implementation, but it would be great to show not tell here if possible - I found the sequential importing and exporting of the `app` a little unclear initially.

Would it be possible for application layers to export an array of routes and/or functions, which can then be used here by `app`? It looks as if Express does support this (docs: https://expressjs.com/en/4x/api.html#app.use).

For example - 
```ts
import express from 'express';
import init from './init';
import cookieless from './cookieless';
import secure from './secure';
import error from './error';

const app = express();
app.use(init)
app.use(cookieless)
app.use(secure)
app.use(error)

export default app;
```

 What do you think?